draft-ietf-capport-architecture.xml

```diff
@@ -444,6 +444,11 @@
                 Device and the User Equipment.
              </t>
           </list>
+
+        Future specifications of a Captive Portal Signaling Protocol will not provide mechanisms for
+        communicating selective blocking. That is, the protocol will not provide mechanisms to block
+        some portions of the external network, but not others. Note that this does not preclude a
+        captive portal deployment from using a walled garden.
```

Comment:
This is a strange statement to make.  We can't really make commitments for future versions of us (who might be entirely different people, even).  I think that all we can do is express the goal of this version.

Perhaps this should be as simple as "The Captive Portal Signaling Protocol does not provide any means of indicating that the network prevents access to some destinations.  The intent is to rely on the web portal to communicate local network policies."

Reply:
Yeah, I struggled with this one; I wasn't comfortable claiming that either. :) I think I misunderstood the request in the issue. I like your suggestion.